src/commands/teardown.rs

```diff
@@ -104,23 +106,30 @@ impl Teardown {
                                         "no network address provided",
                                     )
                                 })?;
-                                let td = TeardownPortForward {
-                                    network: network.clone(),
+                                let spf = SetupPortForward {
```

Comment:
If we're refactoring already, maybe rename this so SetupPortForward isn't used to tear down port forwarding. `PortForwardConfig` maybe?

Reply:
+1